### PR TITLE
Release notes for beta-20170223

### PR DESCRIPTION
**Pull request by bdarnell**
cockroachdb/cockroach#13717

---
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/docs/1111)

### CODE REVIEW

File: beta-20170223.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+### Bug Fixes
+
+- Fixed a bug in error handling that could causes ranges to get stuck if an error occurred during a rebalance. [#13605](https://github.com/cockroachdb/cockroach/pull/13605)
+- Retry table lease errors during schema changes, preventing them from getting stuck. [#13606](https://github.com/cockroachdb/cockroach/pull/13606)
+- Inter-node connections are now closed on heartbeat timeouts, improving the handling of network partitions. [#13654](https://github.com/cockroachdb/cockroach/pull/13654)
+
+### Performance Improvements
+
+- Metadata for queries using unqualified table names is not cached, speeding up these queries by 130%. [#13635](https://github.com/cockroachdb/cockroach/pull/13635)
```
Comment (mberhault) [inline beta-20170223.md]: is this supposed to be `is now cached`?

File: beta-20170223.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-beta-20170223.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-beta-20170223.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+</div>
+
+### SQL Language Changes
+
+- Fixed parsing of interval constants with precision modifiers such as `INTERVAL '5' YEARS`. [#13702](https://github.com/cockroachdb/cockroach/pull/13702)
+- String literals containing `true`, `false`, or their synonyms (but not placeholders or other expressions of type string) can now be used where a `BOOL` is required. [#13685](https://github.com/cockroachdb/cockroach/pull/13685)
+- The underlying implementation of the `DECIMAL` type has been replaced. This changes the default rounding behavior of `DECIMAL` types, which is now consistent with PostgreSQL. [#13551](https://github.com/cockroachdb/cockroach/pull/13551)
```
Comment (mjibson) [inline beta-20170223.md]: It may be good to note here that the results of some calculations may change, since that would surprise users.

**Comment by jseldess** [comment]
We didn't update the `version`, `build_time`, and `start_time` strings for this release and the previous release. I probably should rework the docs release instructions. In any case, updating the strings here: https://github.com/cockroachdb/docs/pull/1120

**Comment by bdarnell** [comment]
Sorry, that was my fault. I thought I knew what was needed without looking at the instructions. Looking at them now, I think the only thing that needs to change is to move the strings.yml section to the end instead of the beginning, since that's when we do it.

**Pull request merged**